PSEUDO API 1.md

```diff
@@ -0,0 +1,217 @@
+
+## XR
+
+	interface XR {
+		Promise<sequence<XRDisplay>> getDisplays();
+
+		attribute EventHandler ondisplayconnect;
+		attribute EventHandler ondisplaydisconnect;
+	};
+
+
+## XRDisplay
+
+	interface XRDisplay : EventTarget {
+		readonly attribute DOMString displayName;
+		readonly attribute boolean isExternal;
+
+		Promise<boolean> supportsSession(XRSessionCreateParametersInit parameters);
+		Promise<XRSession> requestSession(XRSessionCreateParametersInit parameters);
+
+		attribute EventHandler ondeactivate;
+	};
+
+### Todo
+
+- calibration
+- orientation reset
+
+## XRSession
+
+	interface XRSession : EventTarget {
+		readonly attribute XRDisplay display;
+		readonly attribute XRSessionCreateParameters createParameters;
+
+		attribute double depthNear;
+		attribute double depthFar;
+
+		attribute XRLayer layer;
+		attribute Reality reality; // Defaults to most recently used Reality
+
+
+		Reality createEmptyReality();
```

Comment:
See the vr_examples.js for how a VR app would set up using an empty reality.

Perhaps we need some notion of shared and private realities so that a page can indicate that they're creating an empty reality that is really only useful for that one page. I imagine a lot of VR game designs will call for this in order to maintain immersion in the magic circle.